### DISCUSSION TITLE
Redash integration with Snipe-it asset management tool

### DISCUSSION THREAD

**Discussion by faheemhaider1** [getredash/redash#6430] [Q&A] [answered]
Hi, 
I am a fresh grad working as a software engineer. I am facing this issue. I am currently working on an open source inventory management tool known as Snipe-IT. While working with the tool it gives me the option to generate the result for some given categories, but not to a specific query. For instance, an employee has 2 assets under their name. This cannot be presented in a general report, and for that purpose I want to integrate redash to it, so I can run custom queries and generate custom reports out of it. Kindly help in this regards. 
Thanks.

**Comment by guidopetri** [comment, marked as answer]
Hi @faheemhaider1 , thanks for thinking of us! Unfortunately I'm not sure how much help we'll be able to be, since we don't have access to the same things you do :)
